apps/docs/components/ui/table/TypographyTable.tsx

```diff
@@ -38,6 +41,32 @@ function transformDataToTokenData(inputData: Record<string, { name: string; valu
 function groupItemsByPropertiesAndSizes(tokenData: TokenData, itemType: string): GroupedItems {
     const groupedItems: GroupedItems = {} as GroupedItems;
 
+    if (itemType === "overline") {
+        const overlineItem = {} as Record<typeof FontProperties[number], string>;
+
+        FontProperties.forEach(property => {
+            const propertyKey = property;
```

Comment:
not sure why there is an assignation happening here. Why not either rename property to propertyKey, or use property everywhere